pages/LearnerDashboard.html

```diff
@@ -36,11 +36,11 @@ <h2>
     The Subscriptions section:
   </p>
 
-  <img src="static/images/subscriptionsLearnerDashboard.png" style="width: 100%;">
+  <img src="static/images/subscriptionsLearnerDashboard.png" style="width: 100%;" alt="The left-hand navigation bar has 'Subscriptions' highlighted. Three subscriptions are displayed. The 'sort-by' dropdown list reads 'username.'">
```

Comment:
I don't have a visual idea of what a "subscription" is (when you say "Three subscriptions are displayed"; please clarify. Also you refer to "The 'sort-by' dropdown list" but that hasn't been introduced before in the description -- maybe say "The list of subscriptions is sorted by username." or something like that.